ttnn/cpp/ttnn/operations/data_movement/concat/device/concat_device_operation.cpp

```diff
@@ -123,7 +123,7 @@ Tensor concat_impl(std::vector<Tensor> &input_tensors, const std::int64_t dim, c
                             "Current concat implementation requires aligned last dim when concatting on last dim");
                     }
                 }
-                Layout target_layout = Layout::TILE;
+                Layout target_layout = Layout::ROW_MAJOR;
```

Comment:
What's the effect and blast radius of this change?

Reply:
This needs to be ROW_MAJOR, this existing bug is hidden by the auto format later in the code.

Reply:
Seems like after this change for-loop below doesn't make much sense?